<ui version="4.0" >
 <class>AssignHotKey</class>
 <widget class="QDialog" name="AssignHotKey" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Assign a key combination to button</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTextBrowser" name="m_infoText" />
   </item>
   <item>
    <widget class="QLabel" name="m_previewLabel" >
     <property name="text" >
      <string>Key combination</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_previewEdit" >
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AssignHotKey</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AssignHotKey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
